runtime/tchannel_server.go

```diff
@@ -27,15 +27,9 @@ import (
 	"github.com/pkg/errors"
 	"github.com/uber-go/zap"
 	"github.com/uber/tchannel-go"
-)
```

Comment:
Yeah, we can even merge it into `runtime/tchannel/server.go`, which is probably even better?